backend/app/tests/features/briefing_phase.feature

```diff
@@ -0,0 +1,25 @@
+@briefingphase
+Feature: Briefing phase of the game
+  
+  Testing briefing phase for API: https://coldwar-api.onrender.com/docs
+
+  Scenario: user goes into briefing phase with valid 'access_token' and without 'Analyst' card ability
```

Comment:
i think in Scenario keyword we cant use variables - i think you can remove comas from this setension

Reply:
and next - Scenario is used for determine in people underastanding word - what we do in that kase. This string not used for start any tests or determine any variables - variables are determine in Given, tests is in When, Then, And.

Example:

```gherkin
Feature: Fight or flight
  In order to increase the ninja survival rate,
  As a ninja commander
  I want my ninjas to decide whether to take on an
  opponent based on their skill levels

  Scenario: Weaker opponent
    Given the ninja has a third level black-belt
     When attacked by a samurai
     Then the ninja should engage the opponent

  Scenario: Stronger opponent
    Given the ninja has a third level black-belt
     When attacked by Chuck Norris
     Then the ninja should run for his life
```

now, you use same string for all scenarios - determine idea of line of tests in Scenario string